src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -177,9 +176,9 @@ export class DictionaryModal extends React.Component {
           </Modal.Header>
           {errors && <InlineError text={this.errors} />}
           <Modal.Body>
-            <form style={{ marginLeft: '50px', marginRight: '-37%' }}>
-              <div className="row">
-                <div className="col-md-8">
+            <form>
+              <div>
+                <div>
```

Comment:
Now that you've removed the classNames from the divs, are they still necessary?

Reply:
Sorry, I don't understand what you mean by `they`.

Reply:
I don't think so. I will have them removed.

Reply:
I have made the update.